src/transformers/modeling_utils.py

```diff
@@ -1700,6 +1701,12 @@ def save_pretrained(
                 UserWarning,
             )
 
+        if getattr(self, "is_loaded_in_4bit", False):
+            raise NotImplementedError(
+                "You are calling `save_pretrained` on a 4-bit converted model. \
+                                      This is currently not supported"
+            )
```

Comment:
Format is wrong here
```suggestion
            raise NotImplementedError(
                "You are calling `save_pretrained` on a 4-bit converted model. This is currently not supported"
            )
```